packs/agents-of-edgewatch-bestiary/book-2-sixty-feet-under/life-magnet.json

```
@@ -76,7 +76,7 @@
                 "title": "Pathfinder #158: Sixty Feet Under"
             },
             "reset": "<p>The trap deactivates and resets 1 minute after there are no creatures touching the iron bars.</p>",
-            "routine": "<p>(1 action) The trap pulls up to four creatures that have already triggered its Magnetize the Living ability toward one of the far corners of the room, possibly pressing them into the iron bars. If all the iron bars between an affected creature and the magnetic panel remain intact, the creature takes @Damage[(2d10+10)[bludgeoning]] damage with a @Check[fortitude|dc:25]{DC 25 basic Fortitude} save and is restrained on a failure ([[/act escape dc=23]]).</p>\n<p>If an iron bar has been broken, the creature takes @Damage[(1d10+5)[bludgeoning]] damage instead, and if two consecutive bars have been broken the creature takes no damage (but might still be restrained on a failed save).</p>"
+            "routine": "<p>(1 action) The trap pulls up to four creatures that have already triggered its Magnetize the Living ability toward one of the far corners of the room, possibly pressing them into the iron bars. If all the iron bars between an affected creature and the magnetic panel remain intact, the creature takes @Damage[(2d10+10)[bludgeoning]] damage with a @Check[fortitude|dc:25|basic] save and is @UUID[Compendium.pf2e.conditionitems.Item.Restrained] on a failure ([[/act escape dc=23]]).</p>\n<p>If an iron bar has been broken, the creature takes @Damage[(1d10+5)[bludgeoning]] damage instead, and if two consecutive bars have been broken the creature takes no damage (but might still be restrained on a failed save).</p>"
```

```suggestion
            "routine": "<p>(1 action) The trap pulls up to four creatures that have already triggered its Magnetize the Living ability toward one of the far corners of the room, possibly pressing them into the iron bars. If all the iron bars between an affected creature and the magnetic panel remain intact, the creature takes @Damage[(2d10+10)[bludgeoning]] damage with a @Check[fortitude|dc:25|basic|options:inflicts:restrained] save and is @UUID[Compendium.pf2e.conditionitems.Item.Restrained] on a failure ([[/act escape dc=23]]).</p>\n<p>If an iron bar has been broken, the creature takes @Damage[(1d10+5)[bludgeoning]] damage instead, and if two consecutive bars have been broken the creature takes no damage (but might still be restrained on a failed save).</p>"
```
Might be good to start adding these, put those options to use.